### ISSUE TITLE
onImagesLoaded results in images loading multiple times

### ISSUE THREAD

**Issue by RLHawk1** [zurb/foundation-sites#9932]
#### How to reproduce this bug:

1. Use equalizer or something that uses an onImagesLoaded callback with images
2. If not seeing the bug, enable network throttling and disable cache

#### What should happen:
Images in a page should only be loaded once. 

#### What happened instead:
onImagesLoaded is using the following code to detect when images have loaded:
```
if (this.complete || this.readyState === 4 || this.readyState === 'complete') {
  singleImageLoaded();
}
// Force load the image
else {
  // fix for IE. See https://css-tricks.com/snippets/jquery/fixing-load-in-ie-for-cached-images/
  var src = $(this).attr('src');
  $(this).attr('src', src + (src.indexOf('?') >= 0 ? '&' : '?') + new Date().getTime());
  $(this).one('load', function () {
    singleImageLoaded();
  });
}
```

The problem is that when that's called before images have loaded (which, if it's not, it doesn't even need to exist), even on non IE browsers, it switches to using the code in the else block. This causes it to load images two times, doubling the required bandwidth for images.  If the same image is used multiple times on a slow CPU, the Date().getTime() can even return different dates, resulting in the image being loaded even more than twice.

#### Test case:

http://codepen.io/rlhawk1/full/Mpdrao/
![image](https://cloud.githubusercontent.com/assets/1224305/24763051/3663e0f0-1abe-11e7-896c-2f77f2cbb11d.png)

**Comment by kball** [comment]
Good catch. Would you like to take a crack at fixing this @RLHawk1?

**Comment by RLHawk1** [comment]
@kball I can try! Just submitted #9990.

**Comment by kball** [comment]
Fixed in #9990. Will be included in 6.4 and 6.3.2
